examples/05_transformers.ipynb

```diff
@@ -0,0 +1,851 @@
+     "iopub.status.busy": "2021-04-10T16:07:05.849188Z",
+     "iopub.status.idle": "2021-04-10T16:07:06.049023Z",
+     "shell.execute_reply": "2021-04-10T16:07:06.049499Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# example of a univariate series\n",
+    "get_examples(\"pd.Series\", \"Series\")[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of a multivariate series\n",
+    "get_examples(\"pd.DataFrame\", \"Series\")[1]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of a panel in pd-multiindex format\n",
+    "get_examples(\"pd-multiindex\", \"Panel\")[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of the same panel in df-list format\n",
+    "get_examples(\"df-list\", \"Panel\")[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`sktime` supports more in-memory formats, see the \"datatypes and datasets\" tutorial for more details."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.2 General transformer signature - simple transformers<a class=\"anchor\" id=\"section_1_2\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Transformers for `Series` and `Panel` have the same high-level interface. Depending which data type they are more commonly used for, they are found either in the `transformations.series` or `transformations.panel` module. As said, this does not imply a separate interface.\n",
+    "\n",
+    "The most important interface points of transformers are:\n",
+    "\n",
+    "1. construction with parameters, this is as with any other `sktime` estimator\n",
+    "2. fitting the transformer, via `fit`\n",
+    "3. transforming data, via `transform`\n",
+    "4. inverse transforming, via `inverse_transform`\n",
+    "5. updating the transformer, via `update` - not all transformers have this interface point (`update` is currently work in progress, as of v0.8.x, contributions are appreciated)\n",
+    "\n",
+    "We show this in two examples below.\n",
+    "\n",
+    "We will apply transformations to the following `Series` and `Panel` data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sktime.datatypes import get_examples\n",
+    "\n",
+    "# univariate series used in the examples\n",
+    "X_series = get_examples(\"pd.Series\", \"Series\")[0][0:3]\n",
+    "# panel used in the examples\n",
+    "X_panel = get_examples(\"pd-multiindex\", \"Panel\")[0][[\"var_1\"]]\n",
+    "# sub-setting is needed since the example transformer (box-cox)\n",
+    "#   wants univariate and positive data"
```

Comment:
maybe use an example that does not need sub-setting, to avoid having to explain this?

Reply:
yes, I added a fixture just for that recently. Good reminder.

Reply:
... didn't work since needs univariate. Added a positive fixture now.